en/docs/security-announcements/cve-justifications/2025/CVE-2025-66516.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "CVE-2025-66516"
+category: security-announcements
+date: December 12 2025
+---
+
+# CVE-2025-66516
+
+<p class="doc-info">WSO2 Products impacted: no</p>  
+<p class="doc-info">Customer actions required: no</p>  
+---
+
+### REPORTED VULNERABILITY
+
+CVE-2025-66516 [1][2] identifies a vulnerability in the tika-core dependency that allows an attacker to carry out XML External Entity injection via a crafted XFA file inside a PDF.  The entrypoint for the exploitation is through the tika-parser-pdf-module. This CVE covers the same vulnerability as in CVE-2025-54988 [3][4] which was reported against the tika-parser-pdf-module. CVE-2025-66516 redefines an expanded scope for the same CVE-2025-54988 vulnerability due to below reasons.
+
+- Although only the tika-parser-pdf-module was mentioned as affected by CVE-2025-54988 the vulnerability and the fix for it are in tika-core component[5]. Hence the users who only upgraded tika-parser-pdf-module to mitigate CVE-2025-54988 are still vulnerable.
+- In tika 1.x series the PDFParsers classes are in the tika-parsers module and the original CVE-2025-54988 had no mention of it.
```

Comment:
```suggestion
- In tika 1.x series the `PDFParsers` classes are in the `tika-parsers` module and the original CVE-2025-54988 had no mention of it.
```

Reply:
fixed in https://github.com/wso2/docs-security/pull/175/commits/c5c959afd1c9eac9027be8273ff6528fce9db670